yacc/main.c

```diff
@@ -138,7 +138,7 @@ void done(int k)
 
 void onintr(int dummy)
 {
-    done(1);
+    done(128 + dummy);
```

Comment:
Unix convention is that signals cause the program to exit with status 128 + the signal number.